Disable part of test_posix.test_makedev that depends on the NODEV
macro being defined (missing on musl).

Upstream-Status: Submitted [https://github.com/python/cpython/pull/31794]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/Lib/test/test_posix.py
+++ b/Lib/test/test_posix.py
@@ -757,7 +757,7 @@ class PosixTester(unittest.TestCase):
             self.assertRaises((ValueError, OverflowError), posix.makedev, x, minor)
             self.assertRaises((ValueError, OverflowError), posix.makedev, major, x)
 
-        if sys.platform == 'linux':
+        if False:
             NODEV = -1
             self.assertEqual(posix.major(NODEV), NODEV)
             self.assertEqual(posix.minor(NODEV), NODEV)
